### DISCUSSION TITLE
On MacOS I cannot close the front window in Vienna using the keyboard shortcut

### DISCUSSION THREAD

**Discussion by Fedster** [ViennaRSS/vienna-rss#1640] [Q&A] [answered]
Hi All,

I run the latest stable of Vienna on MacOS 13.0.1.  I normally keep Vienna's window closed (I open it, check the RSS feed, than close it, to save screen space) -- closing it using the keyboard requires typing Command-W, which does not normally work.  It does work as expected if I quit Vienna and restart it, but Vienna stops noticing Command-W after a bit, for reasons unknow.  How do I debug it and pass said info on?

**Comment by barijaona** [comment]
Here, Command-W closes a tab (except on the main tab, 'Articles')
To close the window, I have to use Command-Shift-W.

The keyboard binding for these is handled by macOS. I don't know if it is dependent of the localization you use.

**Reply by Fedster** [reply]
Hi, I just have the main tab open (or however that might be called -- basically I do not have tabs).  I have just tried Command-Shit-W and it does not work, but Command-W does (on a freshly restarted Vienna).

**Reply by TAKeanice** [reply]
I think the main tab should forward the call, and close the window if there is no other tab. Or maybe always.

Edit: with should I mean this would be a good behavior, not that it is implemented.

**Comment by barijaona** [comment, marked as answer]
Fixed. Commit 4ffdd65ee19b576bfcb8fb5c3ac4b6508432259c
